contracts/scripts/native_solc8_15_compile_ocr2vrf

```diff
@@ -9,7 +9,7 @@
 ROOT="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; cd ../../ && pwd -P )"
 
 solc-select use 0.8.15
-solc @openzeppelin/=$ROOT/node_modules/@openzeppelin/ --overwrite --optimize --optimize-runs 1000 \
+solc @openzeppelin/=$ROOT/node_modules/@openzeppelin/ --overwrite --optimize --optimize-runs 1000 --metadata-hash none \
```

Comment:
Do you have to re-gen the gethwrappers? Or does this just affect the abi output?

Reply:
I regenned them, yeah it does.

Reply:
I regenned them, yeah it does.